packages/cozy-harvest-lib/src/components/KonnectorSuggestionModal/Illustration.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import AppIcon from 'cozy-ui/transpiled/react/AppIcon'
+
+import preinstallIllustration from '../../assets/preinstall.svg'
+
+const Illustration = ({ alt, iconAlt, app }) => (
+  <div className="u-pos-relative u-mb-half">
+    <img src={preinstallIllustration} alt={alt} />
+    <div
+      className="u-w-2 u-h-2 u-pos-absolute"
+      style={{ top: '18px', left: '39px' }}
```

Comment:
ok I think I have my previous answer... We should at least have a css file to have a few classname... We should not write inline style ;)